Vorticity Display Crash

Selecting vorticity display in the results page on a Samsung A51 crashes the app.

Caused by: java.lang.NullPointerException: Attempt to get length of null array
        at com.onrpiv.uploadmedia.pivFunctions.PivFunctions.createColorMapBitmap(PivFunctions.java:686)
        at com.onrpiv.uploadmedia.Experiment.ViewResultsActivity.createVorticityBitmap(ViewResultsActivity.java:565)
        at com.onrpiv.uploadmedia.Experiment.ViewResultsActivity.displayVortImage(ViewResultsActivity.java:540)

--- a/app/src/main/java/com/onrpiv/uploadmedia/Experiment/ViewResultsActivity.java
+++ b/app/src/main/java/com/onrpiv/uploadmedia/Experiment/ViewResultsActivity.java
@@ -385,7 +385,7 @@ public class ViewResultsActivity extends AppCompatActivity implements PositionCa
 
         // Vorticity
         if (settings.vortMapChanged && settings.getVortDisplay()) {
-            displayVortImage(correlationMaps.get(settings.getVecOption()));
+            displayVortImage();
         } else if (!settings.getVortDisplay()) {
             vorticityImage.setVisibility(View.INVISIBLE);
             vorticityImage.invalidate();
@@ -536,8 +536,8 @@ public class ViewResultsActivity extends AppCompatActivity implements PositionCa
         vectorFieldImage.setVisibility(View.VISIBLE);
     }
 
-    private void displayVortImage(PivResultData pivResultData) {
-        Bitmap bmp = createVorticityBitmap(pivResultData);
+    private void displayVortImage() {
+        Bitmap bmp = createVorticityBitmap(multiPass);
         vorticityImage.setImageBitmap(bmp);
         vorticityImage.setVisibility(View.VISIBLE);
     }

